### DISCUSSION TITLE
Sync with EGS on macOS

### DISCUSSION THREAD

**Discussion by asmaloney** [Heroic-Games-Launcher/HeroicGamesLauncher#2158] [Q&A] [answered]
macOS 12.6.1
Heroic 2.5.1

The sync settings simply says **Sync with Installed Epic Games**, but it's unclear which folder we should point at. (The _Help_ button doesn't have any useful info either.)

I have tried both the folder containing the games themselves and the _~/Library/Application Support/Epic_ directory (to mirror what the Steam seems to have set itself to).

I can't get it to recognize any installed games.

What directory should I be pointing this to?

Thanks!

**Comment by imLinguin** [comment]
If you clear the checkbox and switch tabs back and forth, path should be automatically detected for you

**Comment by asmaloney** [comment]
Thanks Paweł.

I'm not sure what checkbox you're referring to. This is what I have (Heroic 2.5.2 on macOS 12.6.3):

<img width="544" alt="Screen Shot 2023-01-28 at 11 49 27 AM" src="https://user-images.githubusercontent.com/391371/215278710-08814b91-ea63-47ae-bf3b-7fbe9fcc5b23.png">

I can't "Sync" without putting in a path - hence my original question: what directory should this point to?

**Comment by imLinguin** [comment, marked as answer]
Ah, sorry I missunderstood the comment. 

This for Wine Prefix path, so when you have a Windows version of Epic Games Store if I am not mistaking.

~~In terms of importing from Mac native client, I'm not sure about it~~

EDIT: Just checked with `legendary` (the cli tool we use to interact with Epic) 
```
Importing/Exporting installed games from/to the Epic Games Launcher (unsupported for macOS version of EGL)
```
So you likely need to import installed games manually (when clicking install on a game)

**Reply by asmaloney** [reply]
Thanks. I don't think this has anything to do with wine. I have a bunch of macOS games already installed through the official Epic store. I want Heroic to use them instead of downloading them all again.

Unless this is an extremely poorly worded setting (it doesn't mention wine anywhere), that's what it seems to be for 😄 

Edit: uh... ok. I would suggest this setting be removed from the macOS client then. It also kinda makes Heroic less useful for me - I'm not going to do it manually for all my games 😞

**Reply by flavioislima** [reply]
> Thanks. I don't think this has anything to do with wine. I have a bunch of macOS games already installed through the official Epic store. I want Heroic to use them instead of downloading them all again.
> 
> Unless this is an extremely poorly worded setting (it doesn't mention wine anywhere), that's what it seems to be for 😄 
> 
> Edit: uh... ok. I would suggest this setting be removed from the macOS client then. It also kinda makes Heroic less useful for me - I'm not going to do it manually for all my games 😞 

You can import the games manually going to the install dialog and clicking import. But would need to be done one by one ofc. Not way to batch import all on mac for now.
